src/Drawer/Drawer.spec.js

```diff
@@ -134,6 +152,11 @@ describe('<Drawer />', () => {
             <h1>Hello</h1>
           </Drawer>,
         );
+        wrapper.update();
```

Comment:
This looks bug-prone. What about using a `afterEach` hook?

Reply:
What does look bug-prone here?